makefile

```diff
@@ -1,5 +1,15 @@
 MOCK=
+LUA=
 
 .PHONY: test
 test:
-	 find ./test -name "*.lua" | xargs -I {} ./test/tools/run.sh {} $(MOCK)
\ No newline at end of file
+	find ./test -name "*.lua" | LUA=$(LUA) xargs -I {} ./test/tools/run.sh {} $(MOCK)
+
+image:
+	docker build ./test/tools -t umegaya/lua-aws-test
+
+ci:
+	docker run --rm -ti -v `pwd`:/project -w /project -e AWS_ACCESS_KEY=$(AWS_ACCESS_KEY) -e AWS_SECRET_KEY=$(AWS_SECRET_KEY) -e AWS_DEFAULT_REGION=$(AWS_DEFAULT_REGION) -e EC2_URL=$(EC2_URL) umegaya/lua-aws-test bash -c "make test MOCK=$(MOCK) LUA=$(LUA)"
```

Comment:
Can we add the `--init` option or something since otherwise, it seems `ctrl-c` signal is not forwarded to the test script?

Reply:
thanks! I haven't noticed the options is added. fixed.